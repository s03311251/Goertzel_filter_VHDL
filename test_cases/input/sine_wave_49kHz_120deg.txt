3BB6
358F
2D61
23F1
1A22
10E1
090B
035E
0062
0060
0356
08FF
10D1
1A11
23E0
2D51
3582
3BAD
3F3D
3FDE
3D7F
385A
30EB
27E4
1E1F
1488
0C05
0562
0140
0003
01C8
0664
0D68
162D
1FDD
2991
325F
3972
3E21
3FFA
3ED1
3AC3
3430
2BB7
2224
185E
0F4F
07D2
029B
0028
00B3
0430
0A4A
126F
1BDB
25AA
2EF1
36D0
3C8A
3F95
3FA7
3CBF
3724
2F5B
2620
1C52
12DC
0AA3
046C
00CD
001D
026C
0784
0EE9
17E9
21AC
2B47
33D2
3A80
3EB0
3FFD
3E48
39BA
32C0
2A03
2055
169F
0DCB
06AC
01F0
0007
0120
0520
0BA7
1418
1DA8
276F
3084
380B
3D50
