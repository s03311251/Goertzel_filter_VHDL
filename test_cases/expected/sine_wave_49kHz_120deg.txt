00042C40
FFFE05A0
